vike/shared/hooks/executeHook.ts

```diff
@@ -16,8 +16,11 @@ function isUserHookError(err: unknown): false | HookLoc {
   return globalObject.userHookErrors.get(err) ?? false
 }
 
-function executeHook<T = unknown>(hookFn: () => T, hookName: HookName, hookFilePath: string): Promise<T> {
-  const { timeoutErr, timeoutWarn } = getTimeouts(hookName)
+function executeHook<T = unknown>(hookFn: () => T, hookName: HookName, hookFilePath: string, setTimeoutErr?: number, setTimeoutWarn?: number): Promise<T> {
+  // Temporary
+  const fallbackTimeouts = getTimeouts(hookName)
+  const timeoutErr = setTimeoutErr ?? fallbackTimeouts['timeoutErr']
+  const timeoutWarn = setTimeoutWarn ?? fallbackTimeouts['timeoutWarn']
 
```

Comment:
I made this temporary since i still don't know how to get config.timeouts before [executeHook()](https://github.com/vikejs/vike/blob/9669be63f33943b38b25b83d6bf23bd792ae977a/vike/node/prerender/runPrerender.ts#L629) call at runPrerender.ts.
Any suggestion @brillout ?